src/neuroconv/tools/roiextractors/roiextractors.py

```diff
@@ -272,16 +273,30 @@ def add_image_segmentation(nwbfile: NWBFile, metadata: dict) -> NWBFile:
 
 
 def add_two_photon_series(
-    imaging, nwbfile, metadata, buffer_size=10, use_times=False, two_photon_series_index: int = 0
+    imaging,
+    nwbfile,
+    metadata,
+    two_photon_series_index: int = 0,
+    use_times=False,
+    buffer_size: Optional[int] = None,
+    iterator_type: Optional[str] = None,
+    iterator_options: Optional[dict] = None,
 ):
     """
     Auxiliary static method for nwbextractor.
 
     Adds two photon series from imaging object as TwoPhotonSeries to nwbfile object.
     """
-
     if use_times:
         warn("Keyword argument 'use_times' is deprecated and will be removed on or after August 1st, 2022.")
+    if buffer_size:
+        warn(
+            "Keyword argument 'buffer_size' is deprecated and will be removed on or after September 1st, 2022."
+            "Specify as a key in the new 'iterator_options' dictionary instead."
+        )
+    iterator_type = iterator_type or "v2"
```

Comment:
```suggestion

```
If we use "v2" as default this line is not needed

Reply:
This is the case for `add_electrical_series`. `v2` is also the default.